navigation/navigation-common/src/commonTest/kotlin/androidx/navigation/NavDeepLinkTest.kt

```diff
@@ -17,7 +17,6 @@
 package androidx.navigation
 
 import androidx.collection.emptyIntList
-import androidx.core.uri.UriUtils
```

Comment:
`import androidx.navigation.NavUriUtils as UriUtils` ?

Reply:
what for?